movies/2018/andhadhun.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "Andhadhun",
+  "year": 2018,
+  "runtime": 150,
+  "categories": [
+    "thriller",
+    "romance"
+  ],
+  "release-date": "2018-10-05",
+  "director": "Sriram Raghavan"
```

you missed out a comma at the end of director name :)
```suggestion
  "director": "Sriram Raghavan",
```